src/lib/openapi/spec/bulk-registration-schema.ts

```diff
@@ -20,29 +22,48 @@ export const bulkRegistrationSchema = {
                     },
                 },
             },
+            example: ['unleash-edge'],
         },
         appName: {
+            description:
+                'The name of the application that is evaluating toggles',
             type: 'string',
+            example: 'Ingress load balancer',
         },
         environment: {
+            description: 'Which environment is the application running in',
             type: 'string',
+            example: 'development',
         },
         instanceId: {
+            description: 'A unique identifier for the application',
             type: 'string',
+            example: 'Random UUID',
         },
         interval: {
+            description:
+                'How often (in seconds) does the application refresh its features',
```

Comment:
I think questions should be rephrased as statements, as suggested here. The same goes for other similar sentences

```suggestion
                'How often (in seconds) the application refreshes its features',
```

Reply:
Agreed, will fix